packages/wp-source/src/libraries/handlers/__tests__/post-type.tests.ts

```diff
@@ -89,7 +89,7 @@ describe("postType", () => {
 describe("post", () => {
   test("doesn't exist in source.post", async () => {
     // Mock Api responses
-    api.get = jest.fn().mockResolvedValueOnce(mockResponse([post1]));
+    api.get = jest.fn().mockResolvedValue(mockResponse([post1]));
```

Comment:
Previously the data was fetched serially for `post`, `page` and `media` (in that order). So, here we previously could assume that after we resolve the first `api.get()` (for post), there will be no more calls to `api.get()` as we already got the data on the first try.

This is not the case anymore as we're fetching data in parallel so `api.get()` will always be called three times no matter what.

The same change had to be done in a couple other places.